corehq/apps/commtrack/models.py

```diff
@@ -193,6 +157,7 @@ class CommtrackConfig(QuickCachedDocumentMixin, Document):
     multiaction_enabled = BooleanProperty()
     multiaction_keyword_ = StringProperty()
 
+    # todo: remove?
     requisition_config = SchemaProperty(CommtrackRequisitionConfig)
     openlmis_config = SchemaProperty(OpenLMISConfig)
```

Comment:
(not part of this PR but it seems like `OpenLMISConfig` might also be able to be removed)